galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -504,3 +504,27 @@ messages:
         type: string
       - name: hosts
         type: string
+
+  - name: "ImageAutoWithoutInjectionWarning"
+    code: IST0146
+    level: Warning
+    description: "Pods and Deployments with `image: auto` should be targeted for injection."
+    template: "%s %s contains `image: auto` but does not match any Istio injection webhook selectors."
+    url: "https://istio.io/latest/docs/reference/config/analysis/ist0146/"
+    args:
+      - name: resourceType
+        type: string
+      - name: resourceName
+        type: string
+
+  - name: "ImageAutoWithoutInjectionError"
+    code: IST0147
+    level: Error
+    description: "Pods and Deployments with `image: auto` should be targeted for injection."
+    template: "%s %s contains `image: auto` but does not match any Istio injection webhook selectors."
+    url: "https://istio.io/latest/docs/reference/config/analysis/ist0146/"
```

Comment:
Not sure why some others have this - but many don't - but it seems wrong that this is required, or at the very least should point to IST0147?